website/docs/api/bigint-money/createBigIntMoney/Description.md

```diff
@@ -0,0 +1,54 @@
+---
+id: Description
+title: description
+hide_title: true
+sidebar_label: Description
+---
+
+# Unit definitions
+
+### BigIntMoneyBase
+
+
+```ts
+
+interface BigIntMoneyBase {
+  getAmount: () => Money["amount"];
+  getCurrency: () => Money["currency"];
+  add(money: BigIntMoneyBase): BigIntMoneyBase;
+  subtract(money: BigIntMoneyBase): BigIntMoneyBase;
+  isSameCurrency(money: BigIntMoneyBase): boolean;
+  equals(money: BigIntMoneyBase): boolean;
+  compare(money: BigIntMoneyBase): 1 | 0 | -1;
+  greaterThan(money: BigIntMoneyBase): boolean;
+  greaterThanOrEqual(money: BigIntMoneyBase): boolean;
+  lessThan(money: BigIntMoneyBase): boolean;
+  lessThanOrEqual(money: BigIntMoneyBase): boolean;
+
+  multiply(
```

Comment:
Plz make it straight

For example 
 divide(
    number: number | string | bigint,
    roundingMode?: RoundingModesType
  ): BigIntMoneyBase;


Converts to oneliner
 divide(number: number | string | biting, roundingMode?: RoundingModesTyp ): BigIntMoneyBase;